ui/packages/catalog-ui-search/src/main/webapp/component/query-selector/query-selector.view.js

```diff
@@ -65,16 +105,16 @@ var QuerySelector = Marionette.LayoutView.extend({
   },
   addQuery: function() {
     if (this.model.canAddQuery()) {
-      var newQuery = new Query.Model()
+      let newQuery = new Query.Model()
```

Comment:
✏️ As trivial as it is, how about `const` instead? And for the others as well? Ideally we build up a reputation around `let` as rarely used so that when it does get used, we are immediately alerted that the assigned variable value is intentionally changing, and likely needs to be changing.

Reply:
This also makes reviewing easier because when I see `const` I'll know to validate variable use instead of use **and** lifecycle.

Reply:
Changed all but one `let` to `const`.